src/Analyzer.JsonRuleEngine.UnitTests/ExpressionTests.cs

```diff
@@ -191,18 +187,6 @@ public void EvaluateInternalGetEvaluation_NullScope_ThrowsException()
                 .Evaluate(null, new Mock<ILineNumberResolver>().Object);
         }
 
```

Comment:
GitHub hid the other test above this.  The scope can be null still, but this test was validating a specific overload (`EvaluateInternal` that took a callback that returned `Results` instead of  `Evaluations`), and that overload no longer exists - `EvaluateInternal` now always uses `Evaluations` in the callback.